src/utils/storage/index.ts

```diff
@@ -19,23 +19,18 @@ export const storage = new Storage(window.localStorage, '')
 
 export const getStoragePrefix = (id = _getChainId()): string => {
   const name = STORAGE_KEYS[id] || id
-  // Legacy ImmortalDB prefix
-  // @TODO: migrate it
   return `_immortal|v2_${name}__`
```

Comment:
We'll be forever haunted by this.

Reply:
Restored the comment. I was actually working on a migration PR but then decided to get rid of async-await first.